foreshadow/config.py

```diff
@@ -20,9 +20,11 @@
     "reducer": {},
 }
 
+_cfg = {}
+
 
 def get_config(base):
-    """Get a configuration file given a root path.
+    """Try to get config data given given a folder path.
```

Comment:
remove second given
'Try to load configuration data from specific folder path.'